payments/trustly/other-features.md

```diff
@@ -0,0 +1,183 @@
+|                  | └─➔&nbsp;`prefillInfo`            | `object`      | Object representing information of what the UI text fields should be populated with                                                                                                                                                                                                                                   |
+|                  | └─➔&nbsp;`firstName`              | `string`      | Prefilled value to put in the first name text box.                                                                                                                                                                                                                                                                    |
+|                  | └─➔&nbsp;`lastName`               | `string`      | Prefilled value to put in the last name text box.                                                                                                                                                                                                                                                                     |
+
+{% include transactions-reference.md api_resource="trustly"
+documentation_section="trustly" %}
+
+{% include callback-reference.md api_resource="trustly" %}
+
+{% include payment-link.md show_3d_secure=false %}
+
+{% include complete-url.md %}
+
+## Description
+
+{% include description.md %}
+
+{% include payee-info.md api_resource="trustly" %}
+
+{% include prices.md api_resource="trustly" %}
+
+{% include settlement-reconciliation.md %}
+
+## Problems
+
+When performing unsuccessful operations, the eCommerce API will respond with a
+problem message. We generally use the problem message `type` and `status` code
+to identify the nature of the problem. The problem `name` and `description` will
+often help narrow down the specifics of the problem.
+
+### Error types from Swedbank Pay Invoice and third parties
+
+All Trustly error types will have the following URI in front of type:
+`https://api.payex.com/psp/errordetail/trustly/<error-type>`
+
+{:.table .table-striped}
+| Type            | Status | Description                   |
+| :-------------- | :----: | :---------------------------- |
+| `externalerror` |  500   | No error code                 |
+| `inputerror`    |  400   | 10 - ValidationWarning        |
+| `inputerror`    |  400   | 30 - ValidationError          |
+| `inputerror`    |  400   | 3010 - ClientRequestInvalid   |
+| `externalerror` |  502   | 40 - Error                    |
+| `externalerror` |  502   | 60 - SystemError              |
+| `externalerror` |  502   | 50 - SystemConfigurationError |
+| `externalerror` |  502   | 9999 - ServerOtherServer      |
+| `forbidden`     |  403   | Any other error code          |
```

Comment:
These error types look a lot like the ones in `common-problem-types.md`?

Reply:
Yes, I have removed these as we dont have this error responses in Trustly